samples/fr/README.md

```diff
@@ -182,3 +182,33 @@
 - Has light overdraft at end of month
 - Has standard expenses: power, telecom, transport, multimedia, gym
 - Has savings
+
+## Account type not supported
+
```

Comment:
For coherency with the rest of the README, adding the language and table would be better:

```suggestion
**Language**: French 🇫🇷

| Format                       | Link                                                                                                                                           |
| ---------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------- |
| Algoan                       | [🔗](https://raw.githubusercontent.com/algoan/fake-open-banking-data/main/samples/fr/<name>.json)                                      |
| Budget Insight               | [🔗](https://raw.githubusercontent.com/algoan/fake-open-banking-data/main/raw-data/budget_insight_v2_0/fr/<name>.json)                 |
| Linxo Connect Direct Account | [🔗](https://raw.githubusercontent.com/algoan/fake-open-banking-data/main/raw-data/linxo_connect_direct_account_api_v3/fr/<name>.json) |
```

Reply:
So far i only added the algoan format, should i add BI and Linxo as well ? @g-ongenae

Reply:
They will be created automatically by the script: https://github.com/algoan/fake-open-banking-data/blob/main/scripts/converter.ts